server/src/main/java/org/opensearch/index/translog/TruncateTranslogAction.java

```diff
@@ -194,7 +194,8 @@ private boolean isTranslogClean(ShardPath shardPath, ClusterState clusterState,
                 shardPath.getShardId(),
                 translogPath,
                 indexSettings,
-                BigArrays.NON_RECYCLING_INSTANCE
+                BigArrays.NON_RECYCLING_INSTANCE,
+                ""
```

Comment:
question - why are we keeping this empty?

Reply:
this tool is not used for remote store enabled clusters. While truncating translog, nodeId is not needed as well .